api/docker-compose.yaml

```diff
@@ -96,9 +96,28 @@ services:
       volumes:
         - gradle-cache:/.gradle
         - .:/w:cached
+        - ~/.config:/.config
       command: wait-for db:3306 -- ./run-bigdata-migrations.sh bigdata local
       env_file:
         - db-cdr/vars.env
+  db-generate-cloudsql-cdr:
+      depends_on:
+        - db
+      build:
+        context: ./src/dev/server
+      user: ${UID}
+      working_dir: /w/db-cdr
+      volumes:
+        - gradle-cache:/.gradle
+        - .:/w:cached
+        - ~/.config:/.config:cached
+        - ~/.gsutil:/.gsutil:cached
+      entrypoint: ["./generate-cdr/generate-cloudsql-cdr.sh"]
```

Comment:
Are you passing in --project, etc. here somehow?

Reply:
Absolutely.  Its in the ruby part.  Adds +args    Any args get passed on when u use entrypoint instead of command.  Dmohs helped there :).   Awesome.  Just works.